### PR TITLE
ja: Format /web/css using Prettier (part 4)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/web/css` folder of the Japanese locale using Prettier.  This is the fourth part.

### CODE REVIEW

File: files/ja/web/css/css_grid_layout/grid_layout_using_named_grid_lines/index.md [hunk 1]
```diff
@@ -355,8 +369,13 @@ repeat 構文には、トラックリストを指定することもでき、繰
 
 ```html
 <div class="wrapper">
-  <div class="item1">col1-start の線 1 から col2-start の線 2 に配置されています</div>
-  <div class="item2">col1-start の線 2 から col1-start と名付けられた線 2 本にまたがって配置されています</div>
+  <div class="item1">
+    col1-start の線 1 から col2-start の線 2 に配置されています
+  </div>
+  <div class="item2">
+    col1-start の線 2 から col1-start と名付けられた線 2
```
Comment (mfuji09) [inline files/ja/web/css/css_grid_layout/grid_layout_using_named_grid_lines/index.md:376]: 2 and 本 shouldn’ｔ be separated.

File: files/ja/web/css/css_grid_layout/grid_template_areas/index.md [hunk 2]
```diff
@@ -325,39 +333,41 @@ slug: Web/CSS/CSS_grid_layout/Grid_template_areas
...
 ```html
 <div class="media">
-    <div class="image"></div>
-    <div class="text">これはメディアオブジェクトの例です。
-      grid-template-areas を使って、メディアオブジェクトの画像部分とテキスト部分を切り替えることができます。
-    </div>
+  <div class="image"></div>
+  <div class="text">
+    これはメディアオブジェクトの例です。 grid-template-areas
+    を使って、メディアオブジェクトの画像部分とテキスト部分を切り替えることができます。
```
Comment (mfuji09) [inline files/ja/web/css/css_grid_layout/grid_template_areas/index.md:369]: `grid-template-areas` and を shouldn't be separated.

File: files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md [hunk 3]
```diff
@@ -52,7 +57,11 @@ original_slug: Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_L
   </nav>
   <article class="content">
     <h1>メイン記事領域</h1>
-    <p>このレイアウトでは、幅が 500 ピクセル以下の画面では、領域をソース順に表示します。グリッドとグリッド上のアイテムの配置を再定義することで、2 列レイアウト、3 列レイアウトへと変化していきます。</p>
+    <p>
+      このレイアウトでは、幅が 500
```
Comment (mfuji09) [inline files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md:61]: 500 and ピクセル shouldn't be separated.

File: files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md [hunk 4]
```diff
@@ -52,7 +57,11 @@ original_slug: Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_L
...
+      ピクセル以下の画面では、領域をソース順に表示します。グリッドとグリッド上のアイテムの配置を再定義することで、2
+      列レイアウト、3 列レイアウトへと変化していきます。
```
Comment (mfuji09) [inline files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md:63]: 2 and 列 shoudn't be separated.

File: files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md [hunk 5]
```diff
@@ -251,7 +276,11 @@ original_slug: Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_L
   </nav>
   <article class="content">
     <h1>メイン記事領域</h1>
-    <p>このレイアウトでは、幅が 500 ピクセル以下の画面では、領域をソース順に表示します。グリッドとグリッド上のアイテムの配置を再定義することで、2 列レイアウト、3 列レイアウトへと変化していきます。</p>
+    <p>
+      このレイアウトでは、幅が 500
+      ピクセル以下の画面では、領域をソース順に表示します。グリッドとグリッド上のアイテムの配置を再定義することで、2
```
Comment (mfuji09) [inline files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md:281]: same to above.

File: files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md [hunk 6]
```diff
@@ -251,7 +276,11 @@ original_slug: Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_L
   </nav>
   <article class="content">
     <h1>メイン記事領域</h1>
-    <p>このレイアウトでは、幅が 500 ピクセル以下の画面では、領域をソース順に表示します。グリッドとグリッド上のアイテムの配置を再定義することで、2 列レイアウト、3 列レイアウトへと変化していきます。</p>
+    <p>
+      このレイアウトでは、幅が 500
```
Comment (mfuji09) [inline files/ja/web/css/css_grid_layout/realizing_common_layouts_using_grids/index.md:280]: same to above.

**Review by mfuji09 — DISMISSED** [review]
Please refer the comments.

**Comment by queengooborg** [comment]
Self-merging as feedback has been addressed and this PR has been sitting around for weeks.

**Pull request merged**